src/features/pages/markets/etf/check-trading/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import * as styles from './styles.module.scss'
+import Flex from 'features/components/atoms/flex-box'
+import { Localize } from 'components/localization'
+import LinkButton from 'features/components/atoms/link-button'
+import Typography from 'features/components/atoms/typography'
+
+const CheckTrading = () => {
+    return (
+        <Flex.Box direction="col" className={styles.item_container}>
```

Comment:
Container.Fixed can be used here...

Reply:
Alright, will add the fixed container here.